docs/deploy-and-review-process.md

```diff
@@ -0,0 +1,59 @@
+# Deploy and review process
+
+This documentation outlines our current processes for how we deploy the latest changes to ethereum.org, and how we prioritize items in our review process.
+
+## Deploy process
+
+Anything merged into the `dev` branch of [ethereum.org](http://ethereum.org) will go live on ethereum.org at the next release. Currently, we deploy the latest changes to [ethereum.org](http://ethereum.org) twice per week.
+
+These are currently:
+
+Monday between 17:00 and 23:00 UTC.
+
+Thursday between 17:00 and 23:00 UTC.
+
+Occasionally, we’ll also do a patch deploy to fix an urgent or widespread issue.
+
+## Review process
+
+### General review process
+
+We use a first-in, first-out system for reviewing pull requests. The longer a pull request has been open, the higher priority it is for our team to review. In some cases—for example, fixing a high-priority issue or merging low-hanging fruit for a deploy—we will stray from this process and use our best judgement to get higher-impact changes deployed more quickly.
+
+We aim to have every new PR merged or closed within 30 days of opening. As outlined in the following sections, different types of pull requests do have different levels of priority, and this may influence how promptly a pull request is acted on.
+
+### Translations
+
+Translation pull requests are considered high priority. These are when the translated and reviewed files from our translation management platform are submitted as a pull request. They are important to our core value of bringing Ethereum education to as many languages as possible. They also are the accumulation of months of work by translators and reviewers, and any delay to this already long process is undesirable.
+
+**Timeline:** PRs should be merged within 1-4 days of opening (by the next release).
+
+### Feature and bug fixes
+
+Any new feature correlated to an approved issue or bug fix is high-priority. Similar to translators, code contributors dedicate a large amount of their time to improving [ethereum.org](http://ethereum.org) as a resource and addressing their efforts quickly is desirable. All bug fixes should be handled swiftly, but new features that were not discussed, requested or approved do fall under this timeline.
```

Comment:
> but new features that were not discussed, requested or approved do fall under this timeline.

Not sure I understand the phrasing here... if they were not discussed, they _do_ fall under this timeline?

Reply:
```suggestion
Any new feature correlated to an approved issue or bug fix is high-priority. Similar to translators, code contributors dedicate a large amount of their time to improving [ethereum.org](http://ethereum.org) as a resource and addressing their efforts quickly is desirable. All bug fixes should be handled swiftly, but new features that were not discussed, requested or approved do not fall under this timeline.
```

Should be this, nice catch